GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs

```diff
@@ -207,5 +208,369 @@ protected override List<ArcDPSEnums.TrashID> GetTrashMobsIDs()
+                    foreach (AbstractCastEvent c in radiantFury)
+                    {
+                        int start = (int)c.Time;
+                        int duration = 2700;
+                        int expectedHitTime = start + duration;
+                        int attackEnd = start + duration;
+                        int endWave = attackEnd + 900;
+                        attackEnd = GetAttackEndByStunTime(log, target, c, duration, attackEnd);
+                        attackEnd = GetAttackEndByDeterminedTime(log, target, c, duration, attackEnd);
+
+                        if (expectedHitTime <= attackEnd)
+                        {
+                            // Shockwave
+                            AddSolarDischargeDecoration(replay, target, attackEnd, endWave, 1200);
+                        }
+                    }
+
+                    // Supernova - Phase Oneshot
+                    var supernova = casts.Where(x => x.SkillId == SupernovaCM).ToList();
+                    foreach (AbstractCastEvent c in supernova)
+                    {
+                        int start = (int)c.Time;
+                        int duration = 75000;
+                        int expectedHitTime = start + duration;
+                        int attackEnd = start + duration;
+                        attackEnd = GetAttackEndByStunTime(log, target, c, duration, attackEnd);
+
+                        replay.Decorations.Add(new CircleDecoration(true, expectedHitTime, 1200, (start, attackEnd), "rgba(255, 0, 0, 0.2)", new AgentConnector(target)));
+                        replay.Decorations.Add(new CircleDecoration(true, 0, 1200, (start, attackEnd), "rgba(255, 0, 0, 0.2)", new AgentConnector(target)));
+                    }
+
+                    // Solar Cyclone
+                    var solarCyclone = casts.Where(x => x.SkillId == SolarCyclone).ToList();
+                    foreach (AbstractCastEvent c in solarCyclone)
+                    {
+                        int start = (int)c.Time;
+                        int duration = 17900;
+                        int interval = 300;
+                        int attackEnd = start + duration;
+
+                        EffectGUIDEvent kick = log.CombatData.GetEffectGUIDEvent(EffectGUIDs.KickGroundEffect);
+                        if (kick != null)
+                        {
+                            IEnumerable<EffectEvent> kickEffects = log.CombatData.GetEffectEventsByEffectID(kick.ContentID).ToList().Where(x => x.Time >= start && x.Time < start + duration);
+                            foreach (EffectEvent kickEffect in kickEffects)
+                            {
+                                replay.Decorations.Add(new RotatedRectangleDecoration(true, 0, 300, (int)target.HitboxWidth, RadianToDegreeF(kickEffect.Orientation.Z), 0, (start, start + interval), "rgba(255, 0, 0, 0.2)", new PositionConnector(kickEffect.Position)));
```

Comment:
Any example as to how it looks like vs how it is supposed to look like? Also shouldn't the decoration be a pie?

Reply:
Perhaps I'm confusing this with something else, I remember this attack being conicals

Reply:
https://youtu.be/QOe4nBTDUp8 at exactly 4:00 you can see which attack im referring to
Also just noticed as i scrolled through the vod that the 4th anomaly attack had 5 kick effects... I need to check that out tomorrow